config/metadata.yml

```diff
@@ -467,15 +467,15 @@ fields:
     label: Rights License
     predicate: DC:license
   
-  masterFilename:
+  primaryFilename:
     definition: "The name of the file (usually attached to this work) that is considered the master digital file to represent the work."
```

Comment:
can you swap 'master' for 'primary' here too?

Reply:
Looks like its not an indexed field. Am I missing something? looked at schema.rb

Reply:
Sounds good, I doubted it was but wanted to be sure. Once 'master' is out of the descriptions I think this is ready.

Reply:
Sorry, definitions, not descriptions. Line 471 & 435